### PR TITLE
 sgt-back-end (deleted all extra files)

### PR DESCRIPTION
**Pull request by PhillipLam89**

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, looking pretty good. Please double check the required status codes in the instructions.

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  if (!Number.isInteger(id) || id < 1 || isNaN(id)) {
+    return res.status(400).json({ error: `GradeId must be a positive integer only...` });
+  }
+  const query = {
+    text: `
+    delete from "grades"
+          where "gradeId" = ${id}
+    returning *;
+  `,
+    value: [id]
```
Comment (thebearingedge) [inline sgt-back-end/index.js:115]: Does `value` also work here or is it supposed to be `values`?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: sgt-back-end/index.js (hunk 1 above)
Comment (thebearingedge) [reply sgt-back-end/index.js:115]: Honest question, i honestly don't know 🤷

Comment (PhillipLam89) [reply sgt-back-end/index.js:115]: it doesn't work if the property name is "values"  when the array has length of 1. I'm not sure why either. I just tried this a bunch of times 
image

Comment (thebearingedge) [reply sgt-back-end/index.js:115]: Interesting!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**